rstudio/c9s-python-3.9/Dockerfile

```diff
@@ -45,6 +45,11 @@ RUN chmod 1777 /var/run/rstudio-server && \
     mkdir -p /usr/share/doc/R
 COPY rsession.conf /etc/rstudio/rsession.conf
 
+# package installation 
+RUN  dnf install -y libsodium-devel.x86_64 && dnf install -y libgit2-devel.x86_64 && dnf install -y libcurl-devel && dnf install -y cmake && dnf install -y flexiblas-*
+RUN R -e "install.packages('Rcpp')"
+RUN R -e 'install.packages(c("tidyverse", "tidymodels", "plumber", "vetiver", "devtools"))'
```

Comment:
There should be a test that tries to (at least) import each of the installed packages. That way we may rest assured that version upgrades did not break basic functionality.

Reply:
I'm thinking that including these packages in some cases causes loading of native .so libraries from the system. That's why I think even import() is a meaningful thing to test

Reply:
Issue for testing exists at https://issues.redhat.com/browse/RHOAIENG-2308, so let's not keep these comments open.